test/07-openqa-label-known-issues.t

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash -xv
```

Comment:
See https://www.shellcheck.net/wiki/SC2096
```suggestion
#!/usr/bin/env bash
set -xv
```
or
```suggestion
#!/usr/bin/bash -xv
```

Reply:
Actually those should both removed. I tried to used them for some debugging. they are leftovers. thanks man

Reply:
fec0624